### PR TITLE
MRNet Single Disease Model

### PR DESCRIPTION
**Pull request by neelabh17**
@vikasgupta-github, please review.

### CODE REVIEW

File: MRNet-Single-Model/README.md [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+# Contributors
+<p > 
+  -- Neelabh Madan   
+ <a href = https://github.com/neelabh17 target='blank'> <img src=https://github.com/edent/SuperTinyIcons/blob/master/images/svg/github.svg height='30' weight='30'/></a>
+<br>
+
+-- Jatin Prakash <a href = https://github.com/bicycleman15 target='blank'> <img src=https://github.com/edent/SuperTinyIcons/blob/master/images/svg/github.svg height='30' weight='30'/></a>
+
+
```
Comment (vikasgupta-github) [inline MRNet-Single-Model/README.md:98]: Please add the course block in the end.

**Comment by neelabh17** [comment]
Done

File: MRNet-Single-Model/README.md (hunk 1 above)
Comment (neelabh17) [reply MRNet-Single-Model/README.md:111]: Done

**Comment by neelabh17** [comment]
@vikasgupta-github, please review

**Pull request merged**